features/FEATURE_UVISOR/includes/uvisor/api/inc/uvisor-lib.h

```diff
@@ -17,6 +17,8 @@
 #ifndef __UVISOR_API_UVISOR_LIB_H__
 #define __UVISOR_API_UVISOR_LIB_H__
 
+#include "includes/uvisor-lib/uvisor-lib.h"
```

Comment:
I can't just remove it, otherwise I get compilation errors. The issue here is without the include, `UVISOR_PRESENT` doesn't get defined, so I get redefinition errors when the compile process reaches this part of the code: https://github.com/ARMmbed/mbed-os/blob/master/features/FEATURE_UVISOR/includes/uvisor/api/inc/unsupported.h#L74

Which it definitely shouldn't be getting to since the K64F is a supported platform.